addons/sale/models/sale_order_line.py

```diff
@@ -316,9 +316,19 @@ def _compute_name(self):
                 context = {'lang': lang}
                 dp_state = line._get_downpayment_state()
                 if dp_state == 'draft':
-                    name = _("%(line_description)s (Draft)", line_description=name)
+                    name = _('Down Payment: %s (Draft)', format_date(self.env, self.create_date.date()))
```

Comment:
Updated to match the existing translation already existing in the invoicing wizard.